examples/patterns/forms/forms-required.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Forms / Required
+category: _patterns
+---
+
+      <form>
+        <div class="p-form-validation is-error">
+          <label for="exampleTextInputError" required class="is-required">Email address</label>
+          <input class="p-form-validation__input" type="text" id="exampleTextInputError" placeholder="e.g joe@bloggs.com">
```

Comment:
Required should be in the input not the label. It's fine to leave the class on the label.

Reply:
Sure thing 👍